docs/browser_support.md

```diff
@@ -0,0 +1,83 @@
+# Browser support
```

Comment:
@Valve What do you think about the document?

Reply:
A preview of the document: https://github.com/fingerprintjs/fingerprintjs/blob/feature/expand-browser-support/docs/browser_support.md

Reply:
for some reason the link returns 404

Reply:
@Valve Because the branch has been deleted. Please try this link: https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/browser_support.md